website/versioned_docs/version-2.0.0-alpha.72/deployment.mdx

```diff
@@ -350,6 +350,38 @@ After your project has been imported, all subsequent pushes to branches will gen
 
 [Render](https://render.com) offers [free static site hosting](https://render.com/docs/static-sites) with fully managed SSL, custom domains, a global CDN and continuous auto-deploy from your Git repo. Get started in just a few minutes by following [Render's guide to deploying Docusaurus](https://render.com/docs/deploy-docusaurus).
 
+## Deploying to Qovery {#deploying-to-qovery}
```

Comment:
not really needed to backport this doc to older versions, we regularly remove older versions